### PR TITLE
CHK-963-removed-old-verificaRPT-call

### PR DESCRIPTION
**Pull request by Fernando-Granato**
#### List of Changes
The verificaRPT call made during the verification phase of an rptId has been deleted.

#### Motivation and Context
During the verification phase of a payment, the source code to handle the call to the verificaRPT node was eliminated by diverting all RptIds to the VerifyPaymentNotice primitive.

#### How Has This Been Tested?
The tests were performed locally

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt [hunk 1]
```diff
@@ -92,116 +78,51 @@ class PaymentRequestsService(
...
-  }
-
-  fun isNodoError(esitoNodoVerificaRPTRisposta: EsitoNodoVerificaRPTRisposta): Boolean {
-    val outcome = esitoNodoVerificaRPTRisposta.esito
-    val ko = StOutcome.KO.value().equals(outcome)
-    return ko &&
-      (RPT_VERIFY_MULTI_BENEFICIARY_RESPONSE_CODE !=
-        (esitoNodoVerificaRPTRisposta.fault?.faultCode))
-  }
+      val paymentRequestInfo: Mono<PaymentRequestInfo>
```
Comment (ciuffagianluca) [inline src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt:87]: Could you declare it at line 93?

Comment (Fernando-Granato) [reply src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt]: Fixed !!!  🚀

File: src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt [hunk 2]
```diff
@@ -92,116 +78,50 @@ class PaymentRequestsService(
...
-                paymentToken = null,
-                idempotencyKey = null,
-                isCart = false))
-        }
-        return@flatMap paymentRequestInfo
-      }
+    Mono.just(rptId).flatMap { request: RptId ->
+      logger.info(
+        "Calling Nodo for VerifyPaymentNotice for get payment info for rptId: [{}]. PaymentContextCode: [{}]",
+        rptId.value,
```
Comment (pietro-tota) [inline src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt:84]: use here the `request` variable from line 81 or remove` request: RptId` altogether

**Review by pietro-tota — APPROVED** [review]
Well done!

**Review by giovanniberti — APPROVED** [review]
🚀

File: src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt (hunk 2 above)
Comment (Fernando-Granato) [reply src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt:84]: Fixed !! 😎

**Pull request merged**